### ISSUE TITLE
Future Request - Starbound support

### ISSUE THREAD

**Issue by JoeGandy** [Austinb/GameQ#127]
Would be greatly appreciated, other than this your system has done everything i could ask for and more!

**Comment by Austinb** [comment]
You have any documentation on the protocol?  Thanks.

**Comment by ulrichblock** [comment]
Hi Austin, you might want to check the pull request #142

**Comment by Austinb** [comment]
Current pending issue is the protocol version for this game is 7.  Not sure if that has any impact.  But the server is not responding the same across multiple servers and from what I can see is not adhering to the source query protocol standard found at https://developer.valvesoftware.com/wiki/Server_queries.  Lots of extra information after what should be the end of the packet.  Also not doing challenge packets to allow access to player and rule information.

Bottom line: need more specs on the expected query response from the game server.
